### ISSUE TITLE
Possibly provide the user a separate option of saving breakpoints?

### ISSUE THREAD

**Issue by joelmoniz** [processing/processing#3356]
At present, all the changes a user has made in the sketch from the last save, as well as any breakpoints modified, are all saved if the user saves the sketch. Instead of this, maybe the user could be asked whether or not (s)he would like to save changes made to the breakpoints since the last sketch? Alternately, maybe an option for whether or not breakpoints are saved by default could be added to the preferences.txt file?

**Comment by benfry** [comment]
Yeah, need to do this. But also requires reworking how pde.properties is handled, because this is the first real state information that's being added, and we need to make sure it doesn't get removed or rewritten when the breakpoints have been added.

Or maybe we put breakpoints in a separate file. Hm...
